src/open_inwoner/pdc/managers.py

```diff
@@ -33,6 +37,49 @@ def visible_for_user(self, user: User):
             return self.filter(visible_for_authenticated=True)
         return self.filter(visible_for_anonymous=True)
 
+    def filter_for_user_with_zaken(self, user: User):
+        """
+        Returns the categories linked to ZaakTypen for which the user has Zaken.
+        """
+        if not getattr(user, "bsn", None):
+            return self
+
+        cases = fetch_cases(user.bsn)
+
+        months_since_last_zaak_per_zaaktype = {}
+        for case in cases:
+            resolve_zaak_type(case)
+
+            duration_since_start = relativedelta(date.today(), case.startdatum)
```

Comment:
@alextreme just to double check, but I assumed that the `relevante zaakperiode` is used with respect to the `startdatum` of the case

Reply:
That is correct